ocs_ci/deployment/vmware.py

```diff
@@ -188,6 +188,10 @@ def add_nodes(self):
             helpers.generate_config_yaml()
             rhel_module = "RHEL_WORKER_LIST"
 
+        if Version.coerce(self.ocp_version) >= Version.coerce("4.9"):
+            with open(self.scale_up_terraform_var, "a") as fd:
```

Comment:
Is there a chance that this variable will be already defined in that file? If so we might rather think of replacing that line, and if not defined we will add it. But if there is no way it will be part of that file in the future I am OK with this approach.

Also thinking about to have this configurable instead of hardcoded version 7 here.  At some point we will support RHEL 8 nodes and this will bee good to have defined in our configuration.

https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/vsphere/upi_1az_rhel_vsan_3m_3w.yaml#L15

So somewhere here we can define:
```
rhel_workers_major_os_version: 7
```

So here we can take this variable and if it will not be specified we will use 7 as default value.
If at some OCP version we will change RHEL version, then we can maybe change the default value in OCP config file.

So you don't need to care about specifying it in particular configuration files, just take default value 7 here but to have option to overwrite it at some point will be good.